_data/accessibility-tests/tooltip.yml

```
@@ -0,0 +1,96 @@
+    summary_additional: |
+      When you view the tooltip and use ANDI or color contrast analyzer to look at the hex colors,
+      contrast between the tooltip component itself and any adjacent background content is at least 3:1.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.11
+  - summary: You can hover over tooltip content.
+    summary_additional: When you move the pointer over an active tooltip, the tooltip stays open.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+  - summary: Active tooltips can be dismissed without moving the mouse or keyboard focus away from the tooltip trigger.
+    summary_additional: |
+      When you press the `escape` key while the tooltip is active,
+      the tooltip closes.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+  - summary: Active tooltips stay open until they are dismissed.
+    summary_additional: Active tooltips close only when you move focus away from the tooltip trigger or press the `escape` key.
+    test_status: pass
+    test_type: general
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.13
+# Zoom/screen magnification tests
+  - summary: Using zoom does not obstruct the tooltip or other content.
+    summary_additional: When you zoom to 200%, the tooltip content remains visible and does not overlap with other content.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.4
+  - summary: Tooltip remains visible and functional when screen is magnified.
+    summary_additional: When you zoom to 200%, you can still read content in the tooltip and surrounding area easily.
+    test_status: pass
+    test_type: zoom
+    version_tested: 3.8.0
+    wcag_criterion: 1.4.10
+# Keyboard navigation tests
+  - summary: Tooltips can be accessed with only a keyboard.
+    summary_additional: When you tab to an element that has a tooltip, the tooltip activates.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.1
+  - summary: The tooltip does not trap the user or move the user around the page unwillingly.
+    summary_additional: When you open a tooltip, you can navigate in and out of the trigger component without difficulty.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.1.2
+  - summary: Focus indicator is clearly visible around trigger element.
+    summary_additional: When you tab into an element with a tooltip, a visible focus indicator appears around that element.
+    test_status: pass
+    test_type: keyboard
+    version_tested: 3.8.0
+    wcag_criterion: 2.4.7
+# Screen reader tests
+  - summary: Screen reader announces tooltip content.
+    summary_additional: |
+      When you focus on the tooltip trigger,
+      the screen reader announces the tooltip content.
+    test_status: pass
```

This is currently marked as a failure on the checklist. Do we want to update this to reflect that?

```suggestion
    test_status: fail
```

---

Good note. We decided in [this conversation](https://github.com/uswds/uswds-site/pull/2646#discussion_r1587813645) and [this conversation](https://github.com/uswds/uswds/issues/5891#issuecomment-2088981813) to change up the test status and summary. I added a comment to this summary in the spreadsheet top request it be updated there as well. @amycole501 and @alex-hull, can you confirm this change and update the spreadsheet to match?

---

I updated the statuses, and like the simplified status for 1.3.1 and 4.1.2